AK/Format.cpp

```diff
@@ -549,8 +549,11 @@ ErrorOr<void> FormatBuilder::put_f80(
 
 #endif
 
-ErrorOr<void> FormatBuilder::put_hexdump(ReadonlyBytes bytes, size_t width, char fill)
+ErrorOr<void> FormatBuilder::put_hexdump(ReadonlyBytes bytes, size_t width, char fill, bool upper_case)
 {
+    constexpr char const* lowercase_lookup = "0123456789abcdef";
+    constexpr char const* uppercase_lookup = "0123456789ABCDEF";
```

Comment:
It's 2022 and this is C++, use a StringView please

Reply:
These are just used as a lookup array
(same as in `convert_unsigned_to_string(...)`)
I will mark them as `char const[]` instead if that is okay with you.